schema.sql

```diff
@@ -57,3 +57,10 @@ CREATE TABLE visits (
   FOREIGN KEY (vet_id) REFERENCES vets (id)
 );
 
+
+--Performance Audit
+/* Add an email column to your owners table */
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
```

Comment:
## Optional
- [ ] I suggest making the index on the case-insensitive version of the email you can do:
``CREATE INDEX ON owners ((lower(email)));``